pkg/importer/writer.go

```diff
@@ -210,8 +243,12 @@ func buildPathString(path string, params []Param) string {
 }
 
 func (w *writer) writeEndpoint(method string, endpoint Endpoint) {
+	responseContentType := ""
 	header := buildRequestHeadersString(endpoint.Params.HeaderParams())
 	body := buildRequestBodyString(endpoint.Params.BodyParams())
+	if len(endpoint.Responses) != 0 {
+		responseContentType = buildResponseContentString(endpoint.Responses)
+	}
 	reqStr := ""
```

Comment:
Same here (I know you didn't write this, but it's good to look out for)

Reply:
Addressed